Asynchronous-Servers-and-Clients-in-Rest_li.md

```diff
@@ -36,7 +36,7 @@ There are two main options available to write async server implementations in Re
 
 ### Async Server Templates
 
-Rest.li also includes templates to make writing async resources slightly easier. There are templates that use `Callback`s, `Promise`s, and `Task`s for each type of Rest.li resource. For example, for collection resources with primitive keys we have `com.linkedin.restli.server.resources.CollectionResourceAsyncTemplate`, `com.linkedin.restli.server.resources.CollectionResourceTaskTemplate`, and `com.linkedin.restli.server.resources.CollectionResourcePromiseTemplate`. There are similar templates for complex key resources, association resources, and simple resources.
+Rest.li also includes templates to make writing async resources slightly easier. There are templates that use `Callback`s and `Task`s for each type of Rest.li resource. For example, for collection resources with primitive keys we have `com.linkedin.restli.server.resources.CollectionResourceAsyncTemplate` and `com.linkedin.restli.server.resources.CollectionResourceTaskTemplate`. There are similar templates for complex key resources, association resources, and simple resources.
```

Comment:
I might be wrong, but I think we recommended to use the `task` one.  Can you check? if yes, we can explicitly mention that.

Reply:
The Promise template is deprecated, so I am decreasing the mentions of that in this document. From the documentation on the templates, it doesn't seem that we specifically recommend task over async. The main difference is whether the user wants to use Parseq and Tasks or a callback. I can change the documentation to specifically recommend Task if thats what we would like to do going forward.